kolibri/plugins/coach/assets/src/composables/quizCreationSpecs.js

```diff
@@ -117,7 +117,7 @@ export const QuizQuestion = {
 export const QuizSection = {
   section_id: {
     type: String,
-    default: () => String(performance.now()),
+    default: () => Math.random().toString(36).substring(7),
```

Comment:
non-blocking, this is not super easy to read

Reply:
Thanks - I added a comment. It's a snippet I've used before to get a random-enough string. Do you think that helps?